### ISSUE TITLE
New container box height

### ISSUE THREAD

**Issue by Ameeko** [aimeos/ai-cms-grapesjs#15]
Hello!

I created a container
![image](https://user-images.githubusercontent.com/61547701/154516688-6806085a-3b89-4369-8371-2177d8f815ec.png)
The height of the container box is very small, it is difficult to insert an item.

Is there a way to make this box higher by default?

**Comment by aimeos** [comment]
You seem to have a monitor with a very high resolution. Can you experiment a bit by adding:
```
.container-xl {
	min-height: 2.5rem !important;
	height: 5vh;
}
```
The first line is already there and you should set the value of `height` to an appropriate value.

**Comment by Ameeko** [comment]
Thanks for the reply.
I copied the CSS code to aimeos.css in my own theme, 
but the admin will definitely take the CSS from the default theme folder in this IFRAME.
![image](https://user-images.githubusercontent.com/61547701/154796995-10892088-6ac4-498d-9548-24a5714c4770.png)

How do I set this IFRAME to use the CSS of my own theme, just like a website?
![image](https://user-images.githubusercontent.com/61547701/154797209-bfd6d782-a74e-4ae6-9761-8c528dfea014.png)

**Comment by aimeos** [comment]
What's the optimal height value now?

The height of the container shouldn't be added to the frontend CSS because there it's likely to be unwanted. Instead, the CSS for the editor must be added here:
https://github.com/aimeos/ai-cms-grapesjs/blob/master/admin/jqadm/themes/custom.js#L636-L670

The GrapesJS code isn't prepared yet to use the theme CSS. To support themes, adding a "data-theme" attribute in the template and pushing the CSS location by JS here would be the best option:
https://github.com/aimeos/ai-cms-grapesjs/blob/master/admin/jqadm/themes/custom.js#L98-L103

**Comment by Ameeko** [comment]
I understand, but
**1) If I set a fixed height for the container, it will always be that height when editing.** 
The elements placed in the container will be higher than the height of the conatiner. 
This is not good. 
Increasing the min-height or padding would be better.

**2) Even if I put it where you wrote, it doesn't affect it**
Only these affect the container element:
![image](https://user-images.githubusercontent.com/61547701/154848670-dbdd5f43-71b3-411e-8c36-cc49af047835.png)

I modified the underlined parts in the /vendor/aimeos/ai-cms-grapesjs/admin/jqadm/themes/custom.js file, but none of them affected it:
![image](https://user-images.githubusercontent.com/61547701/154848754-355d76c7-4c32-42f0-908f-5ce809150d4b.png)
